docs/docs-content/clusters/edge/hardware-requirements.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: "Hardware Requirements"
+title: "Hardware Requirements"
+description: "Learn about the hardware requirements for Palette Edge hosts."
+hide_table_of_contents: false
+sidebar_position: 0
+tags: ["edge", "architecture"]
+---
+
+An Edge host must meet the minimum hardware requirements to be deployed successfully. Certain features such as Trusted
+Boot also specific hardware requirements. This page provides general guidance on the minimum hardware requirements for
+your Edge hosts.
+
+Due to variations in hardware and firmware capabilities, in addition to different workloads required by each user, you
+should always verify and evaluate specific devices with Palette Edge as part of solution design. To learn more, contact
+Spectro Cloud sales at sales@spectrocloud.com.
+
+## Minimum Requirements
+
+All Edge hosts must meet the following minimum hardware requirements.
+
+### AMD64 Architecture Devices
+
+| Component | Requirement                                                                                                                                                          |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU       | - Intel: i3, i5, i7, i9, Xeon series <br /> - AMD: Ryzen 3, 5, 7, 9, and Epyc series                                                                                 |
+| CPU Core  | Minimum two cores.                                                                                                                                                   |
+| Memory    | Minimum 8 GB.                                                                                                                                                        |
+| Storage   | Main drive requires a minimum of 100 GB storage to accommodate Operating System (OS), Kubernetes, and leave room for workloads. The main drive must be an SSD drive. |
+
+### ARM64 Architecture Devices
+
+ARM64 support is only verified for the Nvidia Jetson Orin device family.
```

Comment:
If it's verified for the family, should we give minimum requirements for devices in that family? Because right now it seems that any Jetson Orin device would work (unless that't the case, then disregard) 👁️